styles/globals.css

```
@@ -1,9 +1,18 @@
+:root {
+  --main-color: #4067a1;
+  --primary-color: #0000;
+  --secundary-color: #a2b2c9;
+  --background-color: #f5f8ff;
+  --text-button-color: #ffff;
+  --positive-color: #2f9f2f;
+}
```

The shortest style for color codes:

```suggestion
:root {
  --main-color: #4067a1;
  --primary-color: #000;
  --secundary-color: #a2b2c9;
  --background-color: #f5f8ff;
  --text-button-color: #fff;
  --positive-color: #2f9f2f;
}
```